aten/src/ATen/native/mps/operations/Normalization.mm

```diff
@@ -234,7 +234,7 @@ Check if running mean exists (maybe do this check before making graph)
                 MPSGraphTensor* scaledBatchMean = [mpsGraph multiplicationWithPrimaryTensor:batchMeanTensor
                                                                             secondaryTensor:momentumTensor
                                                                                        name:nil];
-                MPSGraphTensor* scaledRunningMean = [mpsGraph multiplicationWithPrimaryTensor:runningMeanTensor
+                MPSGraphTensor* scaledRunningMean = [mpsGraph multiplicationWithPrimaryTensor:castMPSTensor(mpsGraph, runningMeanTensor, input_mps_dtype)
```

Comment:
All these casts shouldn't be necessary. If the initial Ranked placeholder is created with correct data type. We shouldn't be adding any casts here in the code.